src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieType } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: MovieType[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard movie={movie} key={movie.imdbId} />
```

Comment:
add `<li>` tag with `key`

Reply:
Fixed your comment. Why did you have to change it? Was it semantically correct, or to understand how to do things in different ways?

Reply:
yes, if you use a <ul> tag, there should be a <li> tag inside it